### ISSUE TITLE
lunotriquetral (LT) interval widening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10729]
**Preferred term label:**
lunotriquetral (LT) interval widening

**Synonyms**

**Definition (free text, please give PubMed ID)**
Radiographic widening of the space between the lunate and triquetrum bones

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lunotriquetral (LT) ligament injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001201
- label: Lunotriquetral interval widening
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: 
- def: Radiographic widening of the space between the lunate and triquetrum bones.
- comment: This finding may be observed with lunotriquetral ligament injury.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10729
- synonyms:
